### DISCUSSION TITLE
S100101 Will pay for a soldered PCB – I've wasted 20 hours already.

### DISCUSSION THREAD

**Discussion by chrstnmr** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#290] [Q&A]
Hello,

I just wanted to upgrade my LayZ Wifi Remote and ended up destroying it. I've spent so much time on it. Also I have spent hours trying to repair it but I am just bad at soldering.
I need one for the S100101 and I will pay if someone ships me a working board (will also pay the shipping costs). My ESP8266 is already flashed but I need a board that is working with a S100101 pump.

Best regards,

Christian

**Comment by chunkysteveo** [comment]
Where are you based?

**Reply by chrstnmr** [reply]
Germany

**Comment by chunkysteveo** [comment]
![20220601_085403](https://user-images.githubusercontent.com/14663002/171355714-7a9e90d9-801b-4ad4-924b-984746519022.jpg)
Is it this board you are looking to be soldered up? With headers and the level shifter?

**Comment by chunkysteveo** [comment]
Woops sorry - I have a 4 wire pump, I don't have any 6 wire cables, or any way to test it working on a new 6 wire pump. Sorry.

**Comment by visualapproach** [comment]
@chrstnmr I have one extra device I can send. Do your pump need resistors to the dsp?

**Reply by chrstnmr** [reply]
I received one from a person on ebay who recently had some PCBs left. It finally works again. I think my soldering iron is not made for small electronic devices.

@visualapproach Would it be a good idea to open a thread in "Discussion" for Europe and America to connect people who have PCBs left or want to sell theirs maybe? Something like "Marketplace Europe" and "Marketplace America"?

**Reply by visualapproach** [reply]
Okay Thanks for answering. And I guess it wouldn't hurt, but there is not tons of offers in the world wide topic "PCB's somethingsomething " (don't remember the exact name)

**Reply by chrstnmr** [reply]
I agree, but maybe some people have duplicate devices left and want to sell them. I guess there are people looking for a way to connect to these people. However, I do not know how legal that is in terms of Bestway's brand and GitHubs guidelines.

**Reply by visualapproach** [reply]
I don't follow; isn't that discussion topic exactly that? I see the point in giving a hint where in the world you are, but isn't it enough to just state that when posting?

**Reply by chrstnmr** [reply]
Oh wait, there is already a topic? I just got that now. I did not know that 😅 . Then nvm my idea since it is basically what you said😄

**Reply by visualapproach** [reply]
No worries I also get lost between all issues and topics.

**Reply by chrstnmr** [reply]
Thank you for being so active on this GitHub project. I really appreciate it and I can imagine it takes a lot of time and effort. So thumbs up to you 👍 

off-topic: Do you think there will be a possibility for an OTA button to simply update the whole thing via the webinterface?

**Reply by visualapproach** [reply]
You're welcome. It takes a lot of time but comments like this and some coffees sent my way is making it a fun hobby. Thank you. I have thought about making a web update available but haven't had the time to read up on it. I think someone added a function to upload new fw via the web interface. In the 4-wire version iirc but I've never looked into it much. It would require several versions precompiled for every model so that's been a hurdle.
